src/Context.tsx

```diff
@@ -13,6 +14,14 @@ interface PuzzleObjectInterface {
 	}
 }
 
+interface UserObjectInterface {
```

Comment:
We should add our interfaces to another file to clean up our context quite a bit